wenyu-gu/gistviewer-wenyu
# Submit the assessment

This pull request brings in all the requested features.

DevQA instructions:

**Prep**
- Make sure you have the proper node environment set up.
- Make sure you have an internet connection.
- Run `npm i`.
- Run `npm run dev` and access the webpage using the displayed local portal.

**QA Steps**
- Enter a username in the homepage search bar and click "Search".
- Observe the gist list being displayed at the bottom of the page.
- Click on the "Details" button on one of the result items.
- Observe the new page displaying more details of the gist.
